test.js

```diff
@@ -82,10 +82,20 @@ test('should skip extra non-configurable non-writable properties', t => {
 	t.is(wrapper.extra, true);
 });
 
-test('should work with arrow functions', t => {
```

Comment:
Why are you removing the `should work with arrow functions` test?

Reply:
Arrow functions `prototype` is `undefined`. Because of that, it was possible to go around the `prototype` non-configurability with them. However that's not super useful to allow that specific case, so this PR forbids any `prototype` copying altogether. So there's no need to test arrow functions specifically.